test/jdk/java/text/testlib/IntlTest.java

```diff
@@ -123,9 +128,10 @@ protected void run(String[] args) throws Exception
             } catch (IllegalAccessException e) {
                 errln("Can't acces test method " + testMethod.getName());
             } catch (InvocationTargetException e) {
+            	// Log the original exception before calling errln(), which throws RuntimeException
```

Comment:
Please indent only with spaces, like the rest of this file.

Reply:
fixed